src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ViewProductSteps.java

```diff
@@ -0,0 +1,61 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import gov.healthit.chpl.aqa.pageObjects.ViewProductPage;
+
+/**
+ * Class ViewVersionSteps definition.
+ */
+public class ViewProductSteps extends Base {
+
+    /**
+     * Get user to the View Version Page.
+     * @param developerId is Developer ID
+     * @param productId is Product ID
+     **/
+    @Given("^I navigate to the product edit page for product \"([^\"]*)\" of developer \"([^\"]*)\"$")
+    public void editVersionPage(final String productId, final String developerId) {
+        getDriver().get(getUrl() + "#/organizations/developers/" + developerId + "/products/" + productId);
+        WebDriverWait wait = new WebDriverWait(getDriver(), TIMEOUT);
+        wait.until(ExpectedConditions.visibilityOf(ViewProductPage.viewProductPageTitle(getDriver())));
+    }
+
+    /**
+     * Click Edit link of a product.
+     * @param productId is the product version id passed in page objects
+     **/
+    @And("^I click the edit link for product \"(.*)\"$")
+    public void clickEditLinkOfProduct(final String productId) {
+        ViewProductPage.editProductLink(getDriver(), productId).click();
+    }
+
+    /**
+     * Set the product name field to "".
```

Comment:
This still isn't right. You're setting the product name to the value passed in, not the empty string. It happens to be true that right now you're only passing in the empty string, but the description of the code doesn't match what it does